### DISCUSSION TITLE
No alerts when replaying pcap files

### DISCUSSION THREAD

**Discussion by davidjud** [Security-Onion-Solutions/securityonion#12238] [2.4]
### Version

2.4.30

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Standalone

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

100

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I have a problem with showing alerts in the SOC when replaying pcap files (for testing) on my standalone SO instance. There are entries in "Dashboard" and "Hunt" but just not a single Alert. What could be the problem in this case?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
What pcap files are you replaying?

Have you tried using the built-in sensor test? https://docs.securityonion.net/en/2.4/grid.html#icons-in-lower-left-corner
From SOC -> Grid if you select a sensor there is an icon for 'Replaying test traffic' this feature requires internet the first time it runs since it pulls a new docker container, but once it runs it will replay traffic that will trigger alerts.
